src/libraries/System.Diagnostics.Debug/tests/System.Diagnostics.Debug.Tests.csproj

```diff
@@ -7,8 +7,6 @@
     <CompileUsingReferenceAssemblies>false</CompileUsingReferenceAssemblies>
   </PropertyGroup>
   <ItemGroup>
-    <DefaultReferenceExclusion Include="System.Diagnostics.Debug" />
-    <DefaultReferenceExclusion Include="System.Runtime.Extensions" />
     <DefaultReferenceExclusion Include="System.Collections" />
```

Comment:
What's the reason for these and the change?  Is it just to avoid duplicate types?

Reply:
Correct. The default reference exclusion was necessary when these assemblies still contained types and weren't pure facade assemblies because of duplicate types.